### DISCUSSION TITLE
Manual priority setting

### DISCUSSION THREAD

**Discussion by FlameArrow57** [stonepillars/goon-flock#228] [Ideas]
Discussion for allowing the Flockmind to manually set the priority of drones if possible. Discussion version of #195.

Interpretation:

Would be cool to have a menu in the Flockmind control panel that lets you manually set the priorities of all drones, apart from the default settings.

Maybe would be cool too to allow setting the priorities of individual drones.

**Comment by TobleroneSwordfish** [comment]
I think this would be difficult to implement well, especially since our AI is set up to allow state based evaluation to largely overrule weighting in most cases. While I love the rimworld style of task priority, that works better over a long term resource management sim (that lets you pause to figure out why your priorities aren't working) than a short term RTS game.
A better system might be blocking drones from certain tasks entirely, such as stopping them from butchering when they should be fighting and building defenses, or blocking them from replicating when you already have enough.
